packages/currencies-number/src/iso4217/amendments/168/aed.ts

```diff
@@ -2,6 +2,8 @@ import type { Currency } from '@dinero.js/core';
 
 /**
  * United Arab Emirates dirham.
+ *
+ * @public
```

Comment:
I added @public to all these for api-extractor

Reply:
Thanks for tagging me, this library looks great actually :D

Reply:
lol sorry I'll write `@public` next time

Reply:
No problem, I love discovering new stuff like this!

Reply:
@public We love new contributors around here!